react/features/video-menu/components/web/LowerHandButton.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import { IJitsiConference } from '../../../base/conference/reducer';
+import { IconRaiseHand } from '../../../base/icons/svg';
+import { PARTICIPANT_ROLE } from '../../../base/participants/constants';
+import { getLocalParticipant } from '../../../base/participants/functions';
+import { LOWER_HAND } from '../../../base/tracks/constants';
+import ContextMenuItem from '../../../base/ui/components/web/ContextMenuItem';
+import logger from '../../logger';
+import { IButtonProps } from '../../types';
+
+interface IProps extends IButtonProps {
+    conference: IJitsiConference | undefined;
+}
+
+/**
+ * Implements a React {@link Component} which displays a button for notifying certain
+ * participant who raised hand to lower hand.
+ *
+ * @returns {JSX.Element}
+ */
+const LowerHandButton = ({
+    conference,
+    participantID
+}: IProps): JSX.Element => {
+    const { t } = useTranslation();
+    const localParticipant = useSelector(getLocalParticipant);
+    const _isModerator = Boolean(localParticipant?.role === PARTICIPANT_ROLE.MODERATOR);
+
+    const handleClick = useCallback(() => {
+        try {
+            _isModerator && conference?.sendEndpointMessage(
```

Comment:
This button should not be visible for non-moderators. Are we testing that?

Reply:
Hi Saghul, I solved it by adding _isModerator condition in the return part. Please let me know if it is not recommended.